components/roadmap-grid/today-marker-toggle.tsx

```diff
@@ -1,20 +1,30 @@
+import { Button, Skeleton, Text } from '@chakra-ui/react';
 import React from 'react';
-import SvgEyeOpenIcon from '../icons/svgr/SvgEyeOpenIcon';
 import SvgEyeClosedIcon from '../icons/svgr/SvgEyeClosedIcon';
-import { Text, Button } from '@chakra-ui/react';
+import SvgEyeOpenIcon from '../icons/svgr/SvgEyeOpenIcon';
 
 import { setShowTodayMarker, useShowTodayMarker } from '../../hooks/useShowTodayMarker';
 
 
+import { useGlobalLoadingState } from '../../hooks/useGlobalLoadingState';
 import styles from './today-marker.module.css';
 
 export function TodayMarkerToggle() {
   const showTodayMarker = useShowTodayMarker();
+  const globalLoadingState = useGlobalLoadingState();
 
   return (
-    <Button className={showTodayMarker ? styles.todayMarkerToggle : styles.todayMarkerToggleDisabled} onClick={() => setShowTodayMarker(!showTodayMarker)}>
-      { showTodayMarker ? <SvgEyeOpenIcon /> : <SvgEyeClosedIcon /> }
+    <Skeleton isLoaded={!globalLoadingState.get()}>
```

Comment:
Does this really depend on globalLoadingState?

Reply:
but, this component doesn't depend on any of the content globalState is blocking on. The toggle button itself can be visible permanently, as content being loaded or not doesn't change whether it's displayable.